buildkitd_config.go

```diff
@@ -0,0 +1,113 @@
+package task
+
+// This file was fetched from
+// https://github.com/moby/buildkit/blob/dc792e75b9adab05f16fcbdd274b9eaf639348c3/cmd/buildkitd/config/config.go
+// and modified to generate buildkitd config files with
+
+// BuildkitdConfig provides containerd configuration data for the server
+type BuildkitdConfig struct {
+	// Debug bool `toml:"debug"`
+
+	// // Root is the path to a directory where buildkit will store persistent data
+	// Root string `toml:"root"`
+
+	//Entitlements e.g. security.insecure, network.host
+	Entitlements []string `toml:"insecure-entitlements"`
+	// GRPC configuration settings
+	GRPC *GRPCConfig `toml:"grpc"`
+
+	Workers *WorkerConfig `toml:"worker"`
+
+	Registries map[string]RegistryConfig `toml:"registry"`
```

Comment:
We only care about `Registries` for now.  But didn't know if worthwhile to leave all the other structs incase we need to extend ability to write other things to config for buildkitd.